### DISCUSSION TITLE
REVOKE keyword does not exist in syntax highlighting in SQL editor

### DISCUSSION THREAD

**Discussion by Simbaclaws** [supabase/supabase#16950] [Contribute to Supabase]
# Bug report

- [ x ] I confirm this is a bug with Supabase, not with my own application. 
- [ x ] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Whenever you try to use the REVOKE keyword which is valid SQL syntax, Supabase doesn't highlight the keyword in syntax highlighting. But queries using the REVOKE keyword are succesfully executed. Might this keyword not exist within your SQL syntax highlighting?

## To Reproduce

1. Login to your supabase instance
2. Open up the SQL Editor
3. Type in REVOKE
4. REVOKE isn't highlighted blue like GRANT is.

## Expected behavior

I expect REVOKE to come up blue as it is a correct keyword to use

## Screenshots

![Example](https://user-images.githubusercontent.com/12895506/230116531-5ecfa1e6-3a92-45c9-8ce8-2864adbca3e6.png)

## System information

- OS: Windows
- Microsoft Edge
- Cloud version

## Additional context

None

**Comment by GaryAustin1** [comment]
https://github.com/supabase/supabase/issues/13267
https://github.com/supabase/supabase/issues/11706

All related to the syntax checker not being updated for Postgres15.
